### ISSUE TITLE
setting for abstract?

### ISSUE THREAD

**Issue by shinandrew** [VT-vision-lab/VQA_LSTM_CNN#22]
Score reported for abstract images using this code is 65 on CodaLab leaderboard.

Yet, running the code as is (which is pre-set for coco dataset),

I got 55 on validation set of abstract dataset using the evaluation tool provided.

Since the difference is pretty large, I'm assuming that setting (e.g. batch size, iterations, learning rate, layer..etc.) should be quite different from real(coco) dataset.

I was wondering if the setting to achieve the reported score on abstract dataset (i.e. how the code should be modified) can be shared.

**Comment by yash-goyal** [comment]
The VQA model for abstract images, which is getting an accuracy of 65 on Open Ended, is using hand-coded features, the description of which is given on VQA Abstract Open-Ended challenge webpage (http://visualqa.org/aoe.html). Let us know if you require the features / feature extraction code.

**Comment by shinandrew** [comment]
> > Let us know if you require the features / feature extraction code.

Yes, Please!

**Comment by yash-goyal** [comment]
Could you please provide your email address?

**Comment by shinysup** [comment]
andrew@mi.t.u-tokyo.ac.jp 
Thanx!
